UPGRADE.md

```diff
@@ -3,6 +3,16 @@
 _After `5.0.0` all `MINOR` and `MAJOR` version bumps will have upgrade notes
 posted here._
 
+[2024-01-19] 6.x.x to 7.x.x
+-----------------------------
+### Overview
+
+##### Twilio Ruby Helper Library’s major version 7.0.0 is now available. We ensured that you can upgrade to Ruby helper Library 6.0.0 version without any breaking changes
```

Comment:
There is 6.0.0 written at one place. At both places, it should be 7..0.0-rc.x. You can refer [this](https://github.com/twilio/twilio-python/commit/01077998c7b89449717e2ca34040f7694e2fcb8f#diff-434625111321abb225365806085a3753cb854d098ed9b4b6dc2001235c7f7bc4).